src/subscriptions/StreamListCard.js

```diff
@@ -24,10 +24,10 @@ const styles = StyleSheet.create({
 
 type Props = $ReadOnly<{|
   dispatch: Dispatch,
-  auth: Auth,
-  canCreateStreams: boolean,
-  streams: Stream[],
-  subscriptions: Subscription[],
+    auth: Auth,
```

Comment:
Are this changes intensional? Or just formatting changes. 

Just curious how this happened, which IDE are you using?

Reply:
I am using VS Code , I don't know how this bad formatting occured.I am going to redo these changes correctly